Wikipedia/Code/NSURL+WMFLinkParsing.m

```diff
@@ -180,7 +180,13 @@ + (NSURL *)wmf_desktopURLForURL:(NSURL *)url {
 #pragma mark - Properties
 
 - (BOOL)wmf_isWikiResource {
-    return [self.path wmf_isWikiResource];
```

Comment:
That last commit fixed the article link taps. 

Was wondering if it's ok that this logic now diverges from the NSString `wmf_isWikiResource` that it had formerly called? Should that method be updated or should this logic move there?